plugins/hls-rename-plugin/hls-rename-plugin.cabal

```diff
@@ -0,0 +1,51 @@
+cabal-version:      2.4
+name:               hls-rename-plugin
+version:            1.0.0.0
+synopsis:           Rename plugin for Haskell Language Server
+description:
+  Please see the README on GitHub at <https://github.com/haskell/haskell-language-server#readme>
+
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Oliver Madine
+maintainer:         madine.oliver@gmail.com
+category:           Development
+build-type:         Simple
+extra-source-files:
+  LICENSE
+  test/testdata/*.hs
+  test/testdata/*.yaml
+
+library
+  exposed-modules:  Ide.Plugin.Rename
+  hs-source-dirs:   src
+  build-depends:
+    , base                  >=4.12    && <5
+    , containers
+    , ghc
+    , ghc-exactprint
+    , ghcide                >=1.4     && <1.5
+    , hiedb
+    , hls-plugin-api        ^>=1.2
+    , hls-retrie-plugin     >=1.0.1.1
+    , lsp
+    , lsp-types
+    , retrie                >=0.1.1.1
```

Comment:
remove the retrie depends

Reply:
We use retrie for Retrie.ExactPrint. Should I try to avoid this?

Reply:
Are those mostly reexportarlo? If so yes, otherwise keep